README.md

```diff
@@ -1,11 +1,11 @@
 <div align="center">
-   <img src="https://goteleport.com/blog/images/2020/gravitational-is-teleport-header.png" width=750/>
+   <img src="https://goteleport.com/blog/_next/static/media/gravitational-is-teleport-header.6fdf3a52.png" width=750/>
    <div align="center" style="padding: 25px">
-      <a href="https://goteleport.com/docs/">
-      <img src="https://img.shields.io/badge/Teleport-8.0-651FFF.svg" />
+      <a href="https://github.com/gravitational/teleport/releases">
```

Comment:
link to goteleport.com/download instead? might be better for tracking numbers

Reply:
this one https://goteleport.com/teleport/download?